lib/dao.js

```diff
@@ -167,9 +168,9 @@ DataAccessObject.create = function (data, options, cb) {
     }
   }
 
-  cb = cb || noCallback;
   data = data || {};
   options = options || {};
+  cb = cb || (Array.isArray(data) ? noCallback : utils.createPromiseCallback());
```

Comment:
> returning data when called with an array and no callback.

Oh, I missed that, you are of course correct. Don't introduce any breaking change :+1: